WalletWasabi.Tests/RegressionTests/SendTests.cs

```diff
@@ -461,17 +461,17 @@ public async Task SendTestsAsync()
 				// covers:
 				// customchange
 				// feePc > 1
-				var k1 = new Key();
-				var k2 = new Key();
+				var k1 = new Key().GetScriptPubKey(ScriptPubKeyType.Segwit);
+				var k2 = new Key().GetScriptPubKey(ScriptPubKeyType.Segwit);
```

Comment:
We should rename the variables to something like `scp1` `scp2`

Reply:
done